### DISCUSSION TITLE
Scoreboard option

### DISCUSSION THREAD

**Discussion by Jonnigames** [Zrips/CMI#6623] [Feature Requests]
Hey, I thougt it would be pretty cool to have a option to give players a scoreboard. CMI supports Nameplates, Tablist and it would be cool, when Scoreboards will be added to ^^

**Comment by Gyztor** [comment]
honestly would try out TAB (https://www.mc-market.org/resources/20631/) should work since it also has an entire customizable scoreboard that can even go on the bungeecord. (as the plugin can be installed on bungeecord and fully work) it might work out relatively well for you. though having some of the functionalities of this plugin included in CMI would also be really cool too tbh

**Reply by MCTGE** [reply]
I am currently using TAB myself for both the tab menu and the scoreboard, however, I would love to have a CMI scoreboard so I could get rid of 1 more plugin from my server.

**Comment by mat-ha3** [comment]
Yeah a scoreboard would be useful.

**Comment by ghost** [comment]
I think scoreboard is so cool,too.

**Comment by Angelius782** [comment]
yes it would be wonderful ... since simplescore is over and not maintain....... :(
